### ISSUE TITLE
Cannot drag edge to a placeholder of a node that has another erroneous node under

### ISSUE THREAD

**Issue by Akirathan** [enso-org/enso#6987]
[Screencast from 2023-06-07 18-05-48.webm](https://github.com/enso-org/enso/assets/14013887/5e1f9791-3111-493a-85eb-a4204f7cf002)

**Comment by kazcw** [comment]
This can happen due to #6772. Does this occur if you close the project and reopen it, or only after detaching the edge in the same session?

**Comment by Akirathan** [comment]
I cannot reproduce it anymore. I thought that it might be caused by the fact that some of the underlying nodes have attached a data flow error. But apparently not. Closing it as "Won't fix".

**Comment by farmaazon** [comment]
Looking at the screencast, I wonder if the issue wasn't due to proximity of the another node on the left: we have defined "area where icons are visible" and this invisible area may cover the `self` port.
